test/unit/root-element.test.js

```diff
@@ -1,74 +1,70 @@
 /* eslint-disable no-unused-expressions */
 import { fixture, defineCE, assert, oneEvent, html, nextFrame } from '@open-wc/testing';
-import { TemplateElement } from 'src/TemplateElement';
+import { LitTemplateElement } from 'src/TemplateElement';
 
 const lightTag = defineCE(
-    class extends TemplateElement {
-        template() {
-            return html`
-                <button ref="lightRef">Click me</button>
-            `;
-        }
+	class extends LitTemplateElement {
+		template() {
+			return html` <button ref="lightRef">Click me</button> `;
+		}
 
-        events() {
-            return {
-                '[ref=lightRef]': {
-                    click: () => this.dispatch('light-event'),
-                },
-            };
-        }
-    },
+		events() {
+			return {
+				'[ref=lightRef]': {
+					click: () => this.dispatch('light-event'),
```

Comment:
@lukas-schardt what is going on here?! I haven't seen these kind of whitespace changes for a long time. Did we miss something in our eslint/prettier config?!